lib/telemetry_metrics.ex

```diff
@@ -9,6 +9,8 @@ defmodule Membrane.TelemetryMetrics do
 
   @type label() :: list()
 
+  @dialyzer [{:nowarn_function, emit_event?: 1}]
```

Comment:
1. It would be best if you narrowed down the `:nowarn` to a flag disabling the specific warning
2. Such a thing requires a comment explaining why is this disabled